examples/base/water-river-with-wikidata.yaml

```diff
@@ -0,0 +1,23 @@
+properties:
+  theme: base
+  type: water
+  class: river
+  subclass: river
+  names:
+    - common:
+      - value: Rio Santana
+        language: local
+  sourceTags:
+    waterway: river
+  sources:
+    - recordID: w353912370@4
+      property: ""
+      dataset: OpenStreetMap
+  wikidata: Q7420045
```

Comment:
This is a GREAT idea @vcschapp that I agree, we should surface.  We do have the recordId in sources, but that is _distinctly different_.

Reply:
This is really a much broader GERS discussion in my opinion. When you figure out two different datasets contain the same object, how to reflect that? Should this field always contain every possible dataset that can match?

Reply:
Love the discussion. We do have the option of holding this idea back until we have stabilized the GERS discussion.

Reply:
Yes. 
Proposal: Keep it as top-level feature in theme=base for now, and revisit later when we have a clear understanding of how Wikidata IDs and GERS IDs might be interconnected? (Because they should be to a degree).

Reply:
Talked about this in Schema TF meeting today, memorialized at [Schema WG issue 196](https://github.com/OvertureMaps/schema-wg/issues/196).

We agreed the `externalIds` bag-of-IDs property has merit but we lack time to align on it and implement it for now, so the outcome was:

1. For the future [Schema Roadmap](https://wiki.overturemaps.org/x/BQEjAQ), the`externalIds` property is added for later prioritization and discusison.
2. For the current PR (see Rob's notes and minutes from issue 196):
    1. [ ] `wikidata` should remain a top-level property
    2. [ ] But shouldn't be linked to OSM.
    3. [ ] As part of this PR, a shared `wikidata` schema definition should be factored out and placed in `schema/defs.yaml` and reused both in:
       - [ ] here, in the Base theme; and
       - [ ] retro-fitted into the Places theme's `brand` object.